common/models/change.js

```diff
@@ -428,6 +428,8 @@ module.exports = function(Change) {
   Change.getCheckpointModel = function() {
     var checkpointModel = this.Checkpoint;
     if (checkpointModel) return checkpointModel;
+    // FIXME(bajtos) This code creates multiple different models with the same
+    // model name, which is not a valid supported usage of juggler's API.
```

Comment:
I think it this cannot be fixed while preserving backwards compatibility. I have opened a follow-up issue to make the fix in LoopBack 3.0 - see https://github.com/strongloop/loopback/issues/1155.